public/index.html

```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width">
     <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css" />
     <link rel="stylesheet" href="dist/css/app.css">
+    <script type="text/javascript" src="https://js.stripe.com/v2/"></script>
```

Comment:
I believe we are now serving the index file at `index.ejs` from `server/views`. Confirm with Jackson that you should add the script tag there.